deepchem/feat/tests/test_graph_features.py

```diff
@@ -88,6 +88,17 @@ def test_alkane(self):
     assert np.array_equal(deg_adj_lists[5], np.zeros([0, 5], dtype=np.int32))
     assert np.array_equal(deg_adj_lists[6], np.zeros([0, 6], dtype=np.int32))
 
+  def test_per_atom_fragmentation(self):
+    """checks if instantiating featurizer with per_atom_fragmentation=True
+    leads to  as many fragments' features, as many atoms mol has for any mol"""
+    import rdkit.Chem
```

Comment:
It would be useful to add an example of `per_atom_fragmentation` directly to the docstring of `ConvMolFeaturizer` as an example

Reply:
Also, I had some troubles with yapf formatter, locally it passed OK (0.22.0)  but here it failedfrom time to time (as you might have noticed from multiple commits), do you know why it may happen?

Reply:
Hmm, I haven't seen that before. Usually the issue is that locally it isn't actually 0.22 that's installed. If there's a yapf failure later when we're closer to merge I can dive in and see if there's something weird happening